src/tools/perf/lib/libperf.c

```diff
@@ -678,7 +698,7 @@ static ucs_status_t uct_perf_test_check_capabilities(ucx_perf_params_t *params,
     status = uct_perf_test_check_md_support(params, params->recv_mem_type, &md_attr);
     if (status != UCS_OK) {
         return status;
-    }
+    } 
```

Comment:
sorry, just noticed extra space after `} `
please fix it

Reply:
@brminich good catch
fixed